src/aap_eda/settings/default.py

```diff
@@ -696,3 +696,6 @@ def get_rulebook_process_log_level() -> RulebookProcessLogLevel:
 API_PATH_TO_UI_PATH_MAP = settings.get(
     "API_PATH_UI_PATH_MAP", {"/api/controller": "/execution", "/": "/#"}
 )
+
+
+ANSIBLE_LOCAL_TEMP = settings.get("ANSIBLE_LOCAL_TEMP", "/tmp")
```

Comment:
@bzwei I think we should be using the python temp directory function, if the tmpfs is marked as readonly this will still fail.

Reply:
https://docs.python.org/3/library/tempfile.html#tempfile.gettempdir